lib/foreman_dlm/engine.rb

```diff
@@ -39,7 +39,9 @@ class Engine < ::Rails::Engine
             :'api/v2/dlmlocks' => [:destroy]
           }, :resource_type => 'ForemanDlm::Dlmlock'
 
-          permission :view_dlmlock_events, {}, :resource_type => 'ForemanDlm::DlmlockEvent'
+          permission :view_dlmlock_events, {
+            :'dlmlocks/:dlmlock_id/dlmlock_event' => [:index]
```

Comment:
This needs to be the controller path, e.g. ` :'api/v2/dlmlock_events' => [:index]`.

Reply:
Ah!